BuildScript.Nuke/Build.cs

```diff
@@ -18,7 +18,7 @@ public partial class Build : NukeBuild
 #pragma warning disable CS0414
   [Solution]
   private readonly Solution Solution = null!;
-#pragma warning disable CS0414
+#pragma warning restore CS0414
```

Comment:
The `Solution` property is no longer unused, therefor the block can be removed alltogether.